hack/build/build-functest.sh

```diff
@@ -0,0 +1,26 @@
+#Copyright 2018 The CDI Authors.
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+
+set -eo pipefail
+
+script_dir="$(readlink -f $(dirname $0))"
+source "${script_dir}"/common.sh
+
+mkdir -p ${TESTS_OUT_DIR}/
+ginkgo build ${CDI_DIR}/test/functional/controller
+mv ${CDI_DIR}/test/functional/controller/controller.test ${TESTS_OUT_DIR}/
+ginkgo build ${CDI_DIR}/test/functional/importer
+mv ${CDI_DIR}/test/functional/importer/importer.test ${TESTS_OUT_DIR}/
+ginkgo build ${CDI_DIR}/tests/
+mv ${CDI_DIR}/tests/tests.test ${TESTS_OUT_DIR}/
```

Comment:
@copejon  As far as I can see, this is supposed to be e2e test, do you want me to create separate script and target in makefile for it ?

Reply:
@lukas-bednar It seems like kubevirt defines "functional" as "e2e", so the names don't need to change and we don't need an e2e target.

Reply:
Based on other comments, I will keep only lines related to `tests/` directory.